cockroachdb/src/jepsen/cockroach.clj

```diff
@@ -968,6 +968,24 @@
                 {:perf (checker/perf)
                  :details (bank-checker)})}))
 
+(defn bank-test-clean
+  [nodes n initial-balance]
+  (basic-test nodes
+    (dissoc {:name "bank-no-nemesis"
+     ;:concurrency 20
```

Comment:
No comment for now; I am rewriting all these test definitions to make the nemesis a parameter.